src/ServiceDescription/Shopify-v1.php

```diff
@@ -3723,5 +3816,13 @@
                 ]
             ]
         ],
+    ],
+    'models' => [
+        'GetResponse' => [
```

Comment:
Yeah... Actually in latest guzzle you need to set a "model". As I don't have the courage to write models for all response (Shopify payload are too complex) I just did as the doc and created a "default" model. But maybe GetResponse is not good :p. I'm goign to change it to "GenericModel".